tests/base.py

```diff
@@ -298,7 +340,7 @@ def perform_and_verify_table_and_field_selection(self,
                 # Verify all fields within each selected stream are selected
                 for field, field_props in catalog_entry.get('annotated-schema').get('properties').items():
                     field_selected = field_props.get('selected')
-                    print("\tValidating selection on {}.{}: {}".format(
+                    LOGGER.info("\tValidating selection on {}.{}: {}".format(
```

Comment:
Use f'string instead of format, wherever required

Reply:
Fixed tap-tester occurrences.